test/utf8/utf8_console_log_null_embedded.js

```diff
@@ -0,0 +1,13 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+// Test if console.log can handle strings with null embedded in them. Null characters should be skipped, and
+// the remaining characters should be printed. See https://github.com/Microsoft/ChakraCore/pull/4539
+
+console.log("abc\0def");
+console.log("ab\0cd\0ef");
+console.log("\0ab\0\0cd\0ef\0");
+console.log("中\0文");
+console.log("\u2014\u0041\u0000\u{10401}");

```

Comment:
Can you make this file have a string with an actual null character in it, just for testing's sake?

Reply:
Done, and `WScriptJsrt::EchoCallback` can correctly handle actual embedded null characters. Now this code works fine on Windows, but on Linux, `console.log` no longer prints new line, even if there's no embedded null, which causes test failures. I haven't figured out why.